guides/v2.1/install-gde/install/cli/install-cli-uninstall.md

```diff
@@ -16,50 +16,48 @@ functional_areas:
 To update the Magento software:
 
 *	If you installed the software from an archive or if you used 'composer-create-project', use the Component Manager or System Upgrade utilities.
-*	If you are a contributing developer (that is, you used `git clone`), see <a href="{{ page.baseurl }}/install-gde/install/cli/dev_options.html">Contributing developers&mdash;update, reinstall Magento</a>.
+*	If you are a contributing developer (that is, you used `git clone`), see [Contributing developers—update, reinstall Magento]({{ page.baseurl }}/install-gde/install/cli/dev_options.html).
 
-<h2 id="instgde-install-magento-reinstall">Reinstall the Magento software</h2>
+## Reinstall the Magento software {#instgde-install-magento-reinstall}
 This section discusses how to uninstall and then reinstall the Magento software with the latest version.
 
 The way you reinstall the Magento application from the command line depends on your role:
 
-*	If you installed the software from an archive or if you used 'composer-create-project', see <a href="#instgde-install-reinst-update-sys">Reinstall as a system integrator</a>.
-*	If you're a contributing developer (that is, you started using `composer clone`), see <a href="{{ page.baseurl }}/install-gde/install/cli/dev_options.html">Contributing developers&mdash;update, reinstall Magento</a>.
+*	If you installed the software from an archive or if you used 'composer-create-project', see [Reinstall as a system integrator](#instgde-install-reinst-update-sys).
+*	If you're a contributing developer (that is, you started using `composer clone`), see [Contributing developers—update, reinstall Magento]({{ page.baseurl }}/install-gde/install/cli/dev_options.html).
 
 
-<h3 id="instgde-install-reinst-update-sys">Reinstall as a system integrator</h3>
+### Reinstall as a system integrator {#instgde-install-reinst-update-sys}
 To reinstall the Magento software as a system integrator:
 
-1.	Log in to your Magento server as a user with permissions to modify files in the Magento file system (for example, the <a href="{{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html">switch to the Magento file system owner</a>.
+1.	Log in to your Magento server as a user with permissions to modify files in the Magento file system (for example, the [switch to the Magento file system owner]({{ page.baseurl }}/install-gde/prereq/file-sys-perms-over.html).
 2.	Enter the following commands in the order shown:
 
 		cd <your Magento install dir>
 		git pull origin develop
 		php bin/magento setup:uninstall
 
-	<div class="bs-callout bs-callout-info" id="info">
-		<span class="glyphicon-class">
-  			<ul><li>If <code>git pull origin develop</code> fails, see <a href="{{ page.baseurl }}/install-gde/trouble/git/tshoot_git-pull-origin.html">troubleshooting</a>. </li>
-  				<li>To use your existing Magento software version , omit the <code>git pull origin develop</code> command.</li></ul></span>
-	</div>
+	{:.bs-callout .bs-callout-info}
```

Comment:
This callout contains more than one line, so you can't use `{:.bs-callout .bs-callout-info}`. It also contains a Liquid variable `{{ page.baseurl }}`, so it has to remain a `<div>`. 

Replace the origin `<div>` with 

`<div class="bs-callout bs-callout-info" markdown="1">`

Your current lines 52 and 53 will be OK when you add the above line.